File: src/ui/app/root/application/models/Fee/Concession.ts
export interface IFeeConcession {
	concessionId : string;
	zoneId : string;
	sessionId : string;
	programId : string;
	shiftId : string;
	challanTypeId : string;
	fullName : string;
	statusId : number;
	loggerId : string;

}
export interface IFeeConcessionExtended {
	concessionId : string;
	zoneId : string;
	sessionId : string;
	programId : string;
	shiftId : string;
	challanTypeId : string;
	fullName : string;
	statusId : number;
	loggerId : string;
	admissionTypeId : string;

}
export interface IFeeConcessionVM {
    concessionId: string;
    zoneId: string;
    sessionId: string;
    programId: string;
    shiftId: string;
    shiftName: String;
    zoneName: String;
    programName: String;
    sessionName: String;
    challanTypeName: String;
    challanTypeId: string;
    fullName: string;
    statusId: number;
    loggerId: string;
}
export interface CheckValidity {
   
    admissionFormId:string;
    concessionId :string;
    concessionName :string;
    installmentNo :number;
    paidDate :Date;
    scholarshipCriteriaId:string;
}
export interface CheckInstallmentP {
   
    admissionFormId:string;
    installmentNo :number;
    paidDate :Date ;

}

File: src/data/Model/Cms360/Fee/Concession.cs
/*
*   Auther: H.Muhammad Kamran
*   email: hmuhdkamran@gmail.com
*   contact: +92 (313 / 333) 9112 845
*/

using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Cms360.Data.Model
{
    [Table("Concession", Schema = "Fee")]
    public partial class FeeConcession
    {
		[Key]
		[Required]
		public Guid ConcessionId { get; set; }
		[Required]
		public Guid ZoneId { get; set; }
		[Required]
		public Guid SessionId { get; set; }
		[Required]
		public Guid ProgramId { get; set; }
		[Required]
		public Guid ShiftId { get; set; }
		[Required]
		public Guid ChallanTypeId { get; set; }
		[Required]
		public string FullName { get; set; }
		[Required]
		public int StatusId { get; set; }
		[Required]
		public Guid LoggerId { get; set; }

    }
	  [Table("VW_Concession", Schema = "Fee")]
    public partial class FeeConcessionVM
    {
		[Key]
		[Required]
		public Guid ConcessionId { get; set; }
		[Required]
		public Guid ZoneId { get; set; }
		[Required]
		public Guid SessionId { get; set; }
		[Required]
		public Guid ProgramId { get; set; }
		[Required]
		public Guid ShiftId { get; set; }
		[Required]
		public String ShiftName { get; set; }
		[Required]
		public String ZoneName { get; set; }
		[Required]
		public String ProgramName { get; set; }
		[Required]
		public String SessionName { get; set; }
		[Required]
		public String ChallanTypeName { get; set; }
		[Required]
		public Guid ChallanTypeId { get; set; }
		[Required]
		public string FullName { get; set; }
		[Required]
		public int StatusId { get; set; }
		[Required]
		public Guid LoggerId { get; set; }

    }
}